R/module-visualize-network-server.R

```diff
@@ -0,0 +1,176 @@
+networkServer <- function(input, output, session, parent_session, significant) {
```

Comment:
nit: Could you rename this to visualizeNetworkServer?  I think networkServer may confuse people thinking it has something to do with computer networks.